src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -41,7 +41,7 @@ public IConfigurationProvider Build(IConfigurationBuilder builder)
             return new HostJsonFileConfigurationProvider(this, _logger);
         }
 
-        private class HostJsonFileConfigurationProvider : ConfigurationProvider
+        public class HostJsonFileConfigurationProvider : ConfigurationProvider
```

Comment:
This doesn't need to be made public as part of this change.

Reply:
have removed it from this change